httpie/cli/argtypes.py

```diff
@@ -134,7 +134,7 @@ class AuthCredentials(KeyValueArg):
     """Represents parsed credentials."""
 
     def has_password(self) -> bool:
-        return self.value is not None
+        return self.value
```

Comment:
I don't understand this change, why are we returning the password in `has_password`. `value` can be either a string (which is not a bool, check the function's signature) or `None`.

Reply:
My mistake, I have fixed by changing it into ``bool(self.value)``. In some cases I saw, that password is ``None``, but sometimes it's empty string ``''``.